pkgs/build-support/rust/default.nix

```diff
@@ -93,8 +93,10 @@ in stdenv.mkDerivation (args // {
 
   installPhase = args.installPhase or ''
     runHook preInstall
-    mkdir -p $out/bin
-    find target/release -maxdepth 1 -executable -type f -exec cp "{}" $out/bin \;
+    mkdir -p $out/bin $out/lib
+    find target/release -maxdepth 1 -type f -executable ! \( -regex ".*.\(so.[0-9.]+\|so\|a\|dylib\)" \) -print0 | xargs -r -0 cp -t $out/bin
```

Comment:
```diff
- find target/release -maxdepth 1 -type f -executable ! \( -regex ".*.\(so.[0-9.]+\|so\|a\|dylib\)" \) -print0 | xargs -r -0 cp -t $out/bin
+ find target/release -maxdepth 1 -type f -executable ! \( -regex ".*.\(so.[0-9.]\+\|so\|a\|dylib\)" \) -print0 | xargs -r -0 cp -t $out/bin 
```

unless the + is escaped it will be not treated as a regex for this dialect.
Same applies for the line below.

Reply:
This fails locally for me hence why I did not escape the +.

```bash
$ find target/release -maxdepth 1 -regex ".*.\(so.[0-9.]+\|so\|a\|dylib\)" -print0
target/release/libcrypto.so.0.0.1
$ find target/release -maxdepth 1 -regex ".*.\(so.[0-9.]\+\|so\|a\|dylib\)" -print0

```

Reply:
Ah, I tried the negation instead that's why.